python/ray/tune/tune.py

```diff
@@ -230,45 +257,23 @@ def run(run_or_experiment,
             is best used with `ray.init(local_mode=True)`).
         restore (str): Path to checkpoint. Only makes sense to set if
             running 1 trial. Defaults to None.
-        search_alg (Searcher): Search algorithm for optimization.
-        scheduler (TrialScheduler): Scheduler for executing
-            the experiment. Choose among FIFO (default), MedianStopping,
-            AsyncHyperBand, HyperBand and PopulationBasedTraining. Refer to
-            ray.tune.schedulers for more options.
-        with_server (bool): Starts a background Tune server. Needed for
-            using the Client API.
         server_port (int): Port number for launching TuneServer.
-        verbose (int): 0, 1, or 2. Verbosity mode. 0 = silent,
-            1 = only status updates, 2 = status and trial results.
-        progress_reporter (ProgressReporter): Progress reporter for reporting
-            intermediate experiment progress. Defaults to CLIReporter if
-            running in command-line, or JupyterNotebookReporter if running in
-            a Jupyter notebook.
-        resume (str|bool): One of "LOCAL", "REMOTE", "PROMPT", or bool.
-            LOCAL/True restores the checkpoint from the local_checkpoint_dir.
-            REMOTE restores the checkpoint from remote_checkpoint_dir.
-            PROMPT provides CLI feedback. False forces a new
-            experiment. If resume is set but checkpoint does not exist,
+        resume (str|bool): One of "LOCAL", "REMOTE", "PROMPT", "ERRORED_ONLY",
+            or bool. LOCAL/True restores the checkpoint from the
+            local_checkpoint_dir, determined
+            by `name` and `local_dir`. REMOTE restores the checkpoint
+            from remote_checkpoint_dir. PROMPT provides CLI feedback.
+            False forces a new experiment. ERRORED_ONLY resets and reruns
```

Comment:
Does ERRORED_ONLY use local checkpoint directory by default? Is there no need to support both ERRORED_ONLY+LOCAL and ERRORED_ONLY+REMOTE?

Reply:
I think only supporting local is ok for now; remote is just added sugar/nice to have - I’ll push a comment.